project-management/zenhub.md

```diff
@@ -0,0 +1,83 @@
+# Working with ZenHub
+
+[ZenHub](https://zenhub.com/) is "agile project management within Github".
+We use it for all squads in the design team.
+
+Use [app.zenhub.com](https://app.zenhub.com), it's much nicer :).
+
+Current squad boards:
+
+- [Snappy squad tasks](https://app.zenhub.com/workspace/o/ubuntudesign/snappy-squad-tasks/boards)
+- [Web squad](https://app.zenhub.com/workspace/o/ubuntudesign/web-squad/boards)
+- [MAAS design squad](https://app.zenhub.com/workspace/o/ubuntudesign/maas-design-squad/boards)
+- [Juju squad](https://app.zenhub.com/workspace/o/ubuntudesign/juju-squad/boards)
+- [Vanilla squad](https://app.zenhub.com/workspace/o/ubuntudesign/vanilla-squad/boards)
+
+## 1. Iterations
+
+- Each iteration should be a milestone in ZenHub in the format `Iteration [Iteration number]-[enddate].[endmonth]`.
+
+ For example `Iteration 9-20.10` followed by `Iteration 10-3.11`.
+
+## 2. Epics
+
+- Each milestone (iteration) should include all epics to be worked on.
+- Epics should have everyone who will be working on it assigned.
+- Epics should always be in the `Iteration Epics` pipeline (see 4.3).
+- Epics should have a description explaining the goal.
+
+## 3. Issues
+
+- Issues for the current milestone should start in the `Current Iteration` pipeline (see 4.4).
+- There should be only one issues per person in the `In Progress` pipeline (see 4.5).
+- Issues should only have one assignee at any one time. If an issue requires multiple people at different stages it should be handed over and the assignee changed at the appropriate time. When reassigned, issues should be moved back into the `Current Iteration` pipeline.
+- Issues should have an estimate based on [the fibonacci estimation guide](https://github.com/canonical-webteam/practices/blob/master/project-management/fibonacci-estimation-guide.md).
```

Comment:
I think we may want to be more specific here. "Issues added to an iteration", or "Issues added to epics". There are other classes is issues:

- untriaged issues, potentially from the community
- triaged issues in the backlog, not yet added to a sprint
- issues in play that weren't planned going into the iteration
- personal tracking of work

Basically issues can be used for far more than sized, planned work, and I think we should encourage that, because it's better that anything is tracked as  unsized issues than that it's not tracked at all or tracked elsewhere.

We may need a deeper discussion about whether we're allowed to ever add points to issues mid-iteration, as it potentially makes our velocity meaningless.

Reply:
I agree with what you're saying however I feel the changes require some chatty time. I think a follow up issue/ pr should be raised if the rest of the points are still valid

Reply:
I'm happy for a follow-up issue to be raise, but that doesn't mean we can merge this if we disagree. Submissions to practices are supposed to move slowly, precisely so all merged practices can be relied upon. So maybe remove this point and move it to an issue?

Reply:
I didn't realise it was the final point, I thought you meant amend things. All is clear now

Reply:
I've updated this point, see what you think? Still worth an issue/ pr for other classes of issue